README.md

```diff
@@ -1148,9 +1148,9 @@ car.save();
 ```javascript
 class Car {
   constructor() {
-    this.make = 'Honda';
-    this.model = 'Accord';
-    this.color = 'white';
+    this.make = '';
```

Comment:
You're right. Let's pass in the `make`, `model`, and `color` into the constructor.